app/templates/app_menu_list.hbs

```diff
@@ -30,11 +29,13 @@
           {{/link-to}}
         </li>
       {{/if}}
-      <li class="menu-list">
-        {{#link-to 'search_code' (query-params backToNewItem=false) class='menu-list-anchor inventory'}}
-          {{fa-icon 'plus-circle' size="lg"}}&nbsp;&nbsp;{{t "add_inventory_item"}}
-        {{/link-to}}
-      </li>
+      {{#each storageTypes as |storageType|}}
+        <li class="menu-list {{if storageType.disable 'disable'}}">
+          <a href="#" class="menu-list-anchor {{if storageType.disable 'disable'}}" {{action 'createInventory' storageType.name}} disabled={{storageType.disable}}>
```

Comment:
As mentioned above, you could directly call that method from here.
`{{action 'createInventory' storageType.name target=packageService}}`

Reply:
sure, @abulsayyad123 i'll make that change. 
Thanks